extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -37,24 +37,27 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	private _initializeDeferred: Deferred<void> = new Deferred<void>();
 	private _openAsUntitled: boolean;
 	private _bookTrustManager: IBookTrustManager;
+	providerId: string;
```

Comment:
Not needed with the change suggested below - but FYI please always specify visibility modifiers when defining instance level vars.